src/EditorFeatures/Core/Extensibility/Completion/ICompletionPresenterSession.cs

```diff
@@ -9,15 +9,15 @@ namespace Microsoft.CodeAnalysis.Editor
 {
     internal interface ICompletionPresenterSession : IIntelliSensePresenterSession
     {
-        void PresentItems(ITrackingSpan triggerSpan, IList<CompletionItem> items, CompletionItem selectedItem,
-            CompletionItem presetBuilder, bool suggestionMode, bool isSoftSelected);
+        void PresentItems(ITrackingSpan triggerSpan, IList<PresentationItem> items, PresentationItem selectedItem,
+            PresentationItem suggestionModeItem, bool suggestionMode, bool isSoftSelected);
```

Comment:
Why do you needed to separate out 'items' and 'suggesteddModeItem' when PresentationItem has a bool on it saying if it's the suggestion mode item?